### PR TITLE
[Website] Ft Homepage tabview PR clean up

### PR DESCRIPTION
**Pull request by FaithDaka**
#### Summary of Changes (What does this PR do?)
- A branch off the homepage tabview chaotic PR

#### Is this change ready to hit production in its current state?
- [x] Yes
- [ ] No

#### Status of maturity (all need to be checked before merging):
- [x] I consider this code done
- [x] I've tested this locally
- [ ] This branch has been LGTM'ed by a reviewer

#### How should this be manually tested?
* git checkout staging
* git pull origin staging
* git checkout origin/ft-homepage-tabview
* cd website/
* npm install 
* npm run standalone
* View the website in a tablet view

#### What are the relevant tickets?
- [WEB-281]

[WEB-281]: https://airqoteam.atlassian.net/browse/WEB-281?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: website/frontend/src/components/AnalyticsSection/AnalyticsSection.js [hunk 1]
```diff
@@ -11,7 +12,7 @@ const AnalyticsSection = () => (
                 <h1>An interactive air quality analytics platform</h1>
                 <div className="AnalyticsSection-info-aside">
                     <p>Access and visualise real-time and historical air quality information across Africa through our easy-to-use air quality analytics dashboard</p>
-                    <a href="https://platform.airqo.net/login" target="_blank" className="AnalyticsSection-info-aside-btn">Learn more {'-->'}</a>
+                    <a href={NETMANAGER_URL} target="_blank" className="AnalyticsSection-info-aside-btn">Learn more {'-->'}</a>
```
Comment (FaithDaka) [inline website/frontend/src/components/AnalyticsSection/AnalyticsSection.js:15]: Users should be routed to platform according to their environment i.e Staging-website -> staging-netmanager, prod-website -> prod-netmanager

File: website/frontend/src/components/MapSection/MapSection.js [hunk 2]
```diff
@@ -13,7 +13,7 @@ const MapSection = () => {
           <span id='first-pill'><p>Air Quality Map</p></span>
           <h3 className='content-h'>Live air quality insights across Africa</h3>
           <span className='content-p'>
-            <p>Visualize hourly air quality information with a single click, over our growing network across African cities.</p>
+            <p>Visualize hourly air quality information with a single click, over our growing network across African cities</p>
```
Comment (FaithDaka) [inline website/frontend/src/components/MapSection/MapSection.js:16]: Content change

File: website/frontend/src/components/Monitor.js [hunk 3]
```diff
@@ -7,7 +7,7 @@ const Monitor = () => (
             <div className="monitor-content">
                 <div className="label"><span>Air Quality Monitor</span></div>
                 <p className="text-main">High-resolution air quality monitoring network</p>
-                <p className="text-sub">We deploy a high-resolution air quality monitoring network in target urban areas across Africa to increase awareness and understanding of air quality management, provide actionable information and derive actions against air pollution.</p>
+                <p className="text-sub">We deploy a high-resolution air quality monitoring network in target urban areas across Africa to increase awareness and understanding of air quality management, provide actionable information and derive actions against air pollution</p>
```
Comment (FaithDaka) [inline website/frontend/src/components/Monitor.js:10]: Content change

**Review by Baalmart — APPROVED** [review]
All looks legit! PR runs well and file changes are pretty fine and aligned with PR purpose. Number of file changes is also reasonable! Generally, good work!  Thanks @FaithDaka 

In the future, we shall emphasise unit testing in all our repos.

**Review by 123MwanjeMike — APPROVED** [review]
LGTM
Thanks @FaithDaka

**Review by Codebmk — APPROVED** [review]

**Pull request merged**